From e1425efa1589cb97347f3195b018163665d2fc52 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 19 Jul 2011 12:54:57 +0800
Subject: [PATCH 0267/2463] ENGR00153270 imx5_defconfig: disable mxc v4l2
 capture

disable config of mxc v4l2 capture to fix build error issue.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 arch/arm/configs/imx5_defconfig |   31 ++++++++++++++-----------------
 1 files changed, 14 insertions(+), 17 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index 794223b..6264dd2 100755
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Tue May 17 09:57:25 2011
+# Tue Jul 19 12:51:19 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -270,7 +270,7 @@ CONFIG_IMX_HAVE_PLATFORM_IMX_IPUV3=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_TVE=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_VPU=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_DVFS=y
-CONFIG_IMX_HAVE_PLATFORM_SATA_AHCI=y
+CONFIG_IMX_HAVE_PLATFORM_AHCI=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_IIM=y
 CONFIG_IMX_HAVE_PLATFORM_MXC_GPU=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_OCOTP=y
@@ -291,6 +291,7 @@ CONFIG_IMX_HAVE_PLATFORM_IMX_SPDIF=y
 # CONFIG_ARCH_MX3 is not set
 # CONFIG_ARCH_MXC91231 is not set
 CONFIG_ARCH_MX5=y
+# CONFIG_ARCH_MX6 is not set
 CONFIG_ARCH_MX50=y
 CONFIG_ARCH_MX51=y
 CONFIG_ARCH_MX53=y
@@ -1402,21 +1403,12 @@ CONFIG_VIDEO_IR_I2C=y
 # CONFIG_VIDEO_UPD64031A is not set
 # CONFIG_VIDEO_UPD64083 is not set
 # CONFIG_VIDEO_VIVI is not set
-CONFIG_VIDEO_MXC_CAMERA=m
-
-#
-# MXC Camera/V4L2 PRP Features support
-#
-CONFIG_VIDEO_MXC_IPU_CAMERA=y
-# CONFIG_VIDEO_MXC_CSI_CAMERA is not set
+# CONFIG_VIDEO_MXC_CAMERA is not set
 # CONFIG_MXC_CAMERA_MICRON111 is not set
 # CONFIG_MXC_CAMERA_OV2640 is not set
-CONFIG_MXC_CAMERA_OV3640=m
-CONFIG_MXC_CAMERA_OV5640=m
-CONFIG_MXC_CAMERA_OV5642=m
-CONFIG_MXC_IPU_PRP_VF_SDC=m
-CONFIG_MXC_IPU_PRP_ENC=m
-CONFIG_MXC_IPU_CSI_ENC=m
+# CONFIG_MXC_CAMERA_OV3640 is not set
+# CONFIG_MXC_CAMERA_OV5640 is not set
+# CONFIG_MXC_CAMERA_OV5642 is not set
 CONFIG_VIDEO_MXC_OUTPUT=y
 CONFIG_VIDEO_MXC_IPU_OUTPUT=y
 # CONFIG_VIDEO_MXC_IPUV1_WVGA_OUTPUT is not set
@@ -1614,11 +1606,12 @@ CONFIG_SND_SOC=y
 CONFIG_SND_SOC_AC97_BUS=y
 CONFIG_SND_IMX_SOC=y
 CONFIG_SND_MXC_SOC_MX2=y
-CONFIG_SND_SOC_IMX_SGTL5000=y
 CONFIG_SND_MXC_SOC_SPDIF_DAI=y
+CONFIG_SND_SOC_IMX_SGTL5000=y
 CONFIG_SND_SOC_IMX_SPDIF=y
 CONFIG_SND_SOC_I2C_AND_SPI=y
 # CONFIG_SND_SOC_ALL_CODECS is not set
+CONFIG_SND_SOC_MXC_SPDIF=y
 CONFIG_SND_SOC_SGTL5000=y
 # CONFIG_SOUND_PRIME is not set
 CONFIG_AC97_BUS=y
@@ -1977,6 +1970,7 @@ CONFIG_CLKDEV_LOOKUP=y
 #
 CONFIG_MXC_IPU=y
 CONFIG_MXC_IPU_V3=y
+CONFIG_MXC_IPU_V3EX=y
 
 #
 # MXC SSI support
@@ -2037,7 +2031,6 @@ CONFIG_MXC_MC13892_POWER=y
 # MXC VPU(Video Processing Unit) support
 #
 CONFIG_MXC_VPU=y
-# CONFIG_MXC_VPU_IRAM is not set
 # CONFIG_MXC_VPU_DEBUG is not set
 
 #
@@ -2069,6 +2062,10 @@ CONFIG_MXC_VPU=y
 CONFIG_MXC_AMD_GPU=y
 
 #
+# ANATOP_THERMAL
+#
+
+#
 # File systems
 #
 CONFIG_EXT2_FS=y
-- 
1.7.7.4

